cab/urls/search.py

```diff
@@ -1,15 +1,9 @@
 from django.urls import path
-from haystack.views import SearchView, basic_search, search_view_factory
 
-from ..forms import AdvancedSearchForm
-from ..views.snippets import autocomplete
-
-search_view = search_view_factory(view_class=SearchView,
-                                  template='search/advanced_search.html',
-                                  form_class=AdvancedSearchForm)
+from ..views.snippets import advanced_search, autocomplete, basic_search
 
 urlpatterns = [
     path('', basic_search, name='cab_search'),
     path('autocomplete/', autocomplete, name='snippet_autocomplete'),
-    path('advanced/', search_view, name='cab_search_advanced'),
+    path('advanced/', advanced_search, name='cab_search_advanced'),
```

Comment:
I wonder if for the first pass we can leave the old functionality as it is, add an extra URL for the new search, deploy that, and then check it all works before removing the old implementation?

Reply:
Yes that makes more sense. Will amend accordingly 👍